chain/network/src/types.rs

```diff
@@ -458,10 +469,25 @@ pub struct Consolidate {
     pub(crate) peer_protocol_version: ProtocolVersion,
 }
 
+// we can't implement this for Consolidate, because that would violate rust rule not to
```

Comment:
What is *this* that you're referring to in 'we can't implement this'?

Reply:
I replaced this comment with a more descriptive one
```
/// Addr<PeerActor> doesn't implement `DeepSizeOf` waiting for `deepsize` > 0.2.0.
```